docs/readthedocs/source/doc/Friesian/QuickStart/Train_wnd.md

```diff
@@ -0,0 +1,59 @@
+### ** Preprocess and Train Wide and Deep Model Using Movielens Data**
+Step 1. to Step 6. are same as [preprocess and train two tower model](Train_2tower.md)
```

Comment:
Each tutorial should be self-contained, so add relevant key concepts and all the step descriptions in the tutorial.

Reply:
There is no need to add the Two Tower Model tutorial, as the W&D tutorial is a superset.